Doc/library/queue.rst

```diff
@@ -233,22 +230,43 @@ Example of how to wait for enqueued tasks to be completed::
 Terminating queues
 ^^^^^^^^^^^^^^^^^^
 
-:class:`Queue` objects can be made to prevent further interaction by shutting
-them down.
+When no longer needed, :class:`Queue` objects can be wound down
```

Comment:
"wound down"? I did not encounter such expression, is it correct?

Reply:
It is [correct](https://www.merriam-webster.com/dictionary/wind%20down). Can you think of another expression that conveys the same thought?  

We want to convey that the default mode allows a graceful shutdown where gets and task_done's can be fully processed versus a hard shutdown (like `sys.exit()` versus `os._exit`).

Reply:
It is just my bad English. I was confused and was not sure that this is not a typo. Google Translate translates this something like "... can be collapsed" or "... can be terminated or terminated", that is not helpful. I hope this is clearer for other non-English speakers.

I cannot propose other poetic variants. "graceful shutdown" and "hard shutdown" is clear to me, although it is slightly repetitive.